sys/clang-format-diff.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python
```

Comment:
Could you be explicit about which version of python you are invoking? Eg. do `#! /usr/bin/python2`. This is needed for example on Archlinux.

Reply:
Actually, it would be great if you could port this to the latest Python version, it is probably not a big effort.